### PR TITLE
[JBPM-9147] getTaskById does not return formName

### PR DESCRIPTION
**Pull request by gmunozfe**
Added test to retrieve formName from kie-server.
**formName** was missing on _ConvertUtils.convertToTask_ method.

Related to [jbpm#1647](https://github.com/kiegroup/jbpm/pull/1647)

### CODE REVIEW

**Comment by gmunozfe** [comment]
jenkins retest this

**Comment by elguardian** [comment]
Jenkins retest this please.

**Comment by gmunozfe** [comment]
jenkins retest this

**Comment by elguardian** [comment]
@gmunozfe  can you fix these ?

**Comment by gmunozfe** [comment]
It needs this fix [jbpm#1649](https://github.com/kiegroup/jbpm/pull/1649)

**Comment by elguardian** [comment]
Jenkins retest this please.

File: kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/jbpm/FormServiceIntegrationTest.java [hunk 1]
```diff
@@ -389,4 +390,46 @@ public void testGetTaskFormWithWrongContainerId() throws Exception {
...
+            
+            List<TaskSummary> tasks = taskClient.findTasksAssignedAsBusinessAdministrator(USER_ADMINISTRATOR, 0, 10);
+            assertNotNull(tasks);
+            Long taskId = tasks.get(0).getId();
+            
+            TaskInstance ti = taskClient.getTaskInstance(CONTAINER_ID, taskId);
+            assertEquals("HRInterview2", ti.getFormName());
+            
+            taskClient.startTask(CONTAINER_ID, taskId, USER_ADMINISTRATOR);
+            taskClient.completeTask(CONTAINER_ID, taskId, USER_ADMINISTRATOR, new HashMap<>());
```
Comment (jomarko) [inline kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/jbpm/FormServiceIntegrationTest.java:411]: Can't we use `Collections.emptyMap()`

File: kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/jbpm/FormServiceIntegrationTest.java [hunk 2]
```diff
@@ -389,4 +390,46 @@ public void testGetTaskFormWithWrongContainerId() throws Exception {
...
+            tasks = taskClient.findTasksAssignedAsBusinessAdministrator(USER_ADMINISTRATOR, 0, 10);
+            assertNotNull(tasks);
+            taskId = tasks.get(0).getId();
+            
+            ti = taskClient.findTaskById(taskId);
+            assertEquals("Tech Interview", ti.getName());
+            assertEquals("TechInterview", ti.getFormName());
+            
+            taskClient.startTask(CONTAINER_ID, taskId, USER_ADMINISTRATOR);
+            taskClient.completeTask(CONTAINER_ID, taskId, USER_ADMINISTRATOR, new HashMap<>());
```
Comment (jomarko) [inline kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/jbpm/FormServiceIntegrationTest.java:422]: Can't we use `Collections.emptyMap()`?

Comment (gmunozfe) [reply kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/jbpm/FormServiceIntegrationTest.java]: Done, thanks for review!

**Pull request merged**